From 443980ddc82fb40e2e1f9544f2be169bd23dd246 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 17 Jun 2017 16:49:24 -0700
Subject: [PATCH] shell.c:  Fix format not a string literal warning

src/shell.c:695:20: error: format not a string literal and no format arguments [-Werror=format-security]
|      fprintf(stderr,zHelp);
|                     ^~~~~

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/shell.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shell.c b/src/shell.c
index bb46c49..3c6fe0f 100644
--- a/src/shell.c
+++ b/src/shell.c
@@ -692,7 +692,7 @@ static int do_meta_command(char *zLine, struct callback_data *p){
   }else
 
   if( c=='h' && strncmp(azArg[0], "help", n)==0 ){
-    fprintf(stderr,zHelp);
+    fprintf(stderr, "%s", zHelp);
   }else
 
   if( c=='i' && strncmp(azArg[0], "indices", n)==0 && nArg>1 ){
-- 
2.13.1

